### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by gadamc**
This branch updates the READMEs to explicitly match the code samples to the Object Storage flavors offered by IBM. Additionally, changes were made for clarity. 

This PR address Issue https://github.com/ibm-watson-data-lab/ibmos2spark/issues/17

@bassel-zeidan -- please review this, but do not merge. After incorporating your thoughts/comments, I'll make the changes, then apply these changes to the R and Scala READMEs as well. After that, we'll have a final review and then merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+![IBM Object Storage Services](fig/ibm_objectstores.png "IBM Object Storage Services")
 
-Using the `stocator` driver connects your Spark executor nodes directly 
-to your data in object storage.
-This is an optimized, high-performance method to connect Spark to your data. All IBM Apache Spark kernels
-are instantiated with the `stocator` driver in the Spark kernel's classpath. 
-You can also run this locally by installing the [stocator driver](https://github.com/SparkTC/stocator) 
-and adding it to your local Apache Spark kernel's classpath. 
 
+This library only does two things.
```
Comment (gadamc) [inline README.md]: I will move this somewhere else -- too much detail for the top of the page. Perhaps I'll create a "Details" Section and a very very brief example "Usage" section.

File: python/README.md [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
-The package sets Spark Hadoop configurations for connecting to
-IBM Bluemix Object Storage and Softlayer Account Object Storage instances. This packages uses the new [stocator](https://github.com/SparkTC/stocator) driver, which implements the `swift2d` protocol, and is availble
-on the latest IBM Apache Spark Service instances (and through IBM Data Science Experience).
+The `ibmos2park` library facilitates data read/write connections between Apache Spark clusters and the various 
+[IBM Object Storage services](https://console.bluemix.net/catalog/infrastructure/object-storage-group). 
 
+![IBM Object Storage Services](fig/ibm_objectstores.png "IBM Object Storage Services")
+
+
+This library only does two things.
```
Comment (gadamc) [inline python/README.md]: Again, I should move this to a  "Details" section.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,36 @@
 # ibmos2spark
 
-The package sets Spark Hadoop configurations for connecting to 
-IBM Bluemix Object Storage and Softlayer Account Object Storage instances. This packages uses the new [stocator](https://github.com/SparkTC/stocator) driver, which implements the `swift2d` protocol, and is availble
-on the latest IBM Apache Spark Service instances (and through IBM Data Science Experience). 
+The `ibmos2park` library facilitates data read/write connections between Apache Spark clusters and the various 
+[IBM Object Storage services](https://console.bluemix.net/catalog/infrastructure/object-storage-group). 
 
+![IBM Object Storage Services](fig/ibm_objectstores.png "IBM Object Storage Services")
```
Comment (gadamc) [inline README.md]: Need an improved image. This is too blurry. How can I get that?

File: python/README.md [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
-```
 
-### Bluemix IBM Cloud Object Storage (COS)
-The class CloudObjectStorage allows you to connect to an IBM bluemix COS. You can connect to a bluemix COS using api keys
-as follows:
+### Cloud Object Storage 
+
+This is the service described on the **far left** in the image above. This service is also called IBM Bluemix Cloud Object Storage (COS) in various locations. 
+
+A `configuration_name` is required and the `cos_type` keyword argument must be set to `bluemix_cos`.
```
Comment (bassel-zeidan) [inline python/README.md]: Currently, configuration_name is not required. if we change this in a future PR we will update the readme file accordingly on that PR.

File: python/README.md [hunk 5]
```diff
@@ -89,38 +99,56 @@ cos = ibmos2spark.CloudObjectStorage(sc, credentials,
...
-### Softlayer Swift Object Storage
+
+### Object Storage OpenStack Swift (Iaas)
+
+This is the service described in **middle right** pane in the image above (and was previously referred to 
+as Softlayer Swift Object Storage). 
+
+Note below that credentials are not passed in as a dictionary, like in the other implementations. 
+Rather, each piece of information is supplied as a separate, required arguement. 
+Also, the `CloudObjectStorage` class is not used here, but rather a `softlayer` object is instantiated.
```
Comment (bassel-zeidan) [inline python/README.md]: This line is a bit confusing. We don't need to mention CloudObjectStorage here

File: python/README.md [hunk 6]
```diff
@@ -141,10 +169,42 @@ slos = ibmos2spark.softlayer(sc, configuration_name, auth_url, tenant, username,
 data = sc.textFile(slos.url(container_name, object_name))
 ```
 
+### Object Storage OpenStack Swift for Bluemix
+
+This is the service described in **far right** pane in the image above. 
+This was previously referred to as Bluemix Swift Object Storage and is still the default object
+storage service provided in Data Science Experience, as of 12 Oct 2017. Credentials are passed as
```
Comment (bassel-zeidan) [inline python/README.md]: I think there is no point of mentioning what is default on DSX.

Comment (gadamc) [reply python/README.md]: Yeah, you're right that this isn't the place for it.

File: README.md (hunk 1 above)
Comment (gadamc) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (gadamc) [reply README.md]: not done...

File: python/README.md (hunk 2 above)
Comment (gadamc) [reply python/README.md]: done

File: python/README.md (hunk 4 above)
Comment (gadamc) [reply python/README.md]: done.

File: python/README.md (hunk 5 above)
Comment (gadamc) [reply python/README.md]: removed

**Comment by gadamc** [comment]
I've updated the readme's per your suggestions. Please let me know if those changes look okay. 

On second thought, after we're done with the review, I'll merge this into master and then issue new PRs for the R and Scala README updates.

**Pull request merged**